pkg/restapi/operation/operations_test.go

```diff
@@ -1002,7 +1148,22 @@ func Test_writeReadAllDocumentsSuccess(t *testing.T) {
 	})
 }
 
-func createDataVaultExpectSuccess(t *testing.T, op *Operation) {
+func createConfigStoreExpectSuccess(t *testing.T, op *Operation) {
+	err := op.vaultCollection.provider.CreateStore(dataVaultConfigurationStoreName)
+	require.NoError(t, err)
+}
+
+func storeSampleConfigExpectSuccess(t *testing.T, op *Operation) {
+	store, err := op.vaultCollection.provider.OpenStore(dataVaultConfigurationStoreName)
+	require.NoError(t, err)
+
+	err = store.StoreDataVaultConfiguration(&models.DataVaultConfiguration{ReferenceID: testReferenceID},
```

Comment:
Should be passing in a valid VaultID here (i.e. not an unencoded UUID)

Reply:
done